glTF-BinExporter/glTF/GlTFUtils.cs

```diff
@@ -157,5 +157,12 @@ public static void ExportBinary(MemoryStream outStream, IEnumerable<RhinoObject>
 
             outStream.Flush();
         }
+
+        public static int AddAndReturnIndex<T>(this List<T> list, T item)
```

Comment:
💎 🎉  Nice! These kind of helper functions are super welcome to declutter everything a bit.

Reply:
We can cleanup the old ones as we go. It'll probably have to be after we switch to the glTF lib backend.